OpenRA.Mods.Common/Traits/LimitedAmmo.cs

```diff
@@ -17,58 +17,121 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Actor has a limited amount of ammo, after using it all the actor must reload in some way.")]
 	public class LimitedAmmoInfo : ITraitInfo
 	{
+		[Desc("Used to link Armaments to this AmmoPool.")]
+		public readonly string AmmoPoolName = "primary";
+
+		[Desc("How much limited ammo does this pool contain when fully loaded.")]
 		public readonly int Ammo = 0;
 
-		[Desc("Defaults to value in Ammo.")]
-		public readonly int PipCount = 0;
+		[Desc("Initial ammo the actor is created with. Defaults to Ammo.")]
+		public readonly int InitialAmmo = -1;
+
+		[Desc("Defaults to value in Ammo. 0 means no visible pips.")]
+		public readonly int PipCount = -1;
 
+		[Desc("PipType to use for loaded ammo.")]
 		public readonly PipType PipType = PipType.Green;
+
+		[Desc("PipType to use for empty ammo.")]
 		public readonly PipType PipTypeEmpty = PipType.Transparent;
 
-		[Desc("Time to reload measured in ticks.")]
+		[Desc("How much ammo is reloaded after a certain period.")]
+		public readonly int ReloadCount = 0;
+
+		[Desc("How much ammo is reloaded after a certain period.")]
+		public readonly string RearmSound = null;
+
+		[Desc("Time to reload on airfield etc. measured in ticks.")]
 		public readonly int ReloadTicks = 25 * 2;
 
-		public object Create(ActorInitializer init) { return new LimitedAmmo(this); }
+		[Desc("Whether or not ammo is replenished on its own.")]
+		public readonly bool SelfReloads = false;
+
+		[Desc("Time to reload automatically measured in ticks.")]
```

Comment:
Either remove "measured in ticks" or rename this variable, please.
Duplication is unnecessary.